### ISSUE TITLE
The minSdk version should not be declared in the android manifest file

### ISSUE THREAD

**Issue by cguino** [thegrizzlylabs/geniusscan-sdk-demo#36]
Please, remove these lines from **android/src/main/AndroidManifest.xml**:
```
  <uses-sdk
    android:minSdkVersion="19"
    />
```

- Versions 
react native 0.63.4
com.android.tools.build:gradle 4.0.1
@thegrizzlylabs/react-native-genius-scan 3.0.26

- Error: 
```
Caused by: com.android.builder.errors.EvalIssueException: Failed to parse XML in /Users/xxx/AwesomeProjet/node_modules/@thegrizzlylabs/react-native-genius-scan/android/src/main/AndroidManifest.xml
  The minSdk version should not be declared in the android manifest file. You can move the version from the manifest to the defaultConfig in the build.gradle file.
```

- step to reproduce:
```gradlew clean && gradlew assembleRelease```

[more info](https://stackoverflow.com/questions/51795814/the-minsdk-version-should-not-be-declared-in-the-android-manifest-file)

**Comment by guillaume-tgl** [comment]
Thanks for reporting this issue. We've fixed this in the v4 version of our SDK, but not in the v3. I'm going to publish a new version with the fix.

**Comment by guillaume-tgl** [comment]
We've just published v3.0.27 of the ReactNative plugin that should fix this issue. Let me know if I can close this issue.

**Comment by cguino** [comment]
Thank you !
